### PR TITLE
Add link to app::home-page with used container

### PR DESCRIPTION
**Pull request by lowtower**
This PR adds a link to the home-page with the used container similar to the used router and template engine.
As it's not a good idea to inject a complete container instance, the used container is retrieved only by the container's class name, which now is first argument of HomePageHandler.

### CODE REVIEW

File: src/App/src/Handler/HomePageHandler.php [hunk 1]
```diff
@@ -1,65 +1,88 @@
...
-            $data['templateDocs'] = 'http://twig.sensiolabs.org/documentation';
-        } elseif ($this->template instanceof ZendViewRenderer) {
-            $data['templateName'] = 'Zend View';
-            $data['templateDocs'] = 'https://docs.zendframework.com/zend-view/';
-        }
-
-        return new HtmlResponse($this->template->render('app::home-page', $data));
-    }
-}
+<?php
```
Comment (weierophinney) [inline src/App/src/Handler/HomePageHandler.php]: Why are the entire contents of this file changed? Could you check your IDE to make sure it's not changing line endings, please?

Comment (lowtower) [reply src/App/src/Handler/HomePageHandler.php]: Well, it does :-( !!

I'll check and correct it

**Comment by lowtower** [comment]
Okay, I see that the `RequestHandlerInterface` must be met by the `HomePageHandler`, so I cannot change the constructor in the way I thought I could.
So, the tests must fail.

**Comment by lowtower** [comment]
Sorry for confusing You!
I thought it would be nice to have also the container links on the home page when starting with `zend-expressive`.
I hope, I have it acceptable right now!

File: src/App/src/Handler/HomePageHandler.php [hunk 2]
```diff
@@ -38,6 +44,23 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
 
         $data = [];
 
+        if ('Aura\Di\Container' == $this->containerName) {
```
Comment (samsonasik) [inline src/App/src/Handler/HomePageHandler.php]: can use `===` comparison, ref https://3v4l.org/YDr7A

Comment (webimpress) [reply src/App/src/Handler/HomePageHandler.php]: I would go with `switch` in that case

File: test/AppTest/Handler/HomePageHandlerTest.php [hunk 3]
```diff
@@ -15,17 +16,25 @@
 
 class HomePageHandlerTest extends TestCase
 {
+    /** @var ContainerInterface */
```
Comment (webimpress) [inline test/AppTest/Handler/HomePageHandlerTest.php]: It should be `ContainerInterface|ObjectProphecy`.

The same we should have for `$router`.

**Comment by weierophinney** [comment]
Thanks, @lowtower!

**Pull request merged**